barretenberg/cpp/src/barretenberg/translator_vm/translator_proving_key.hpp

```diff
@@ -95,13 +96,12 @@ class TranslatorProvingKey {
     {
         // Check that the Translator Circuit does not exceed the fixed upper bound, the current value 8192 corresponds
         // to 10 rounds of folding (i.e. 20 circuits)
-        if (circuit.num_gates > Flavor::MINIMUM_MINI_CIRCUIT_SIZE) {
+        const size_t shift_offset = 1; // take into acount the left-by-1 shift
+        if (circuit.num_gates + shift_offset > Flavor::TRANSLATOR_VM_FIXED_SIZE) {
```

Comment:
is the initial zero row not included in num_gates? It would be in the case of ultra but not sure how its done for the translator

Reply:
hah you're right, good catch, very not obvious in the circuit builder code tho